examples/sign_verify.rs

```diff
@@ -0,0 +1,35 @@
+extern crate bitcoin_hashes;
+extern crate secp256k1;
+
+use bitcoin_hashes::{sha256, Hash};
+use secp256k1::{All, Error, Message, PublicKey, Secp256k1, SecretKey, Signature};
```

Comment:
Should we use `bitcoin::hashes` and `bitcoin::secp256k1` here to guarantee version compatibility?

Reply:
Oh never mind, this is rust-secp not rust-bitcoin.